addons/web/controllers/json.py

```diff
@@ -262,7 +262,9 @@ def get_default_domain(model, action, context, eval_context):
     for ir_filter in model.env['ir.filters'].get_filters(model._name, action._origin.id):
         if ir_filter['is_default']:
             # user filters, static parsing only
-            default_domain = ast.literal_eval(ir_filter['domain'])
+            domain_str = ir_filter['domain']
+            domain_str = domain_str.replace('uid', str(model.env.uid))
+            default_domain = ast.literal_eval(domain_str)
```

Comment:
Hello @mash-odoo Thank you for the fix!

@kmagusiak  Can you check what would be best this or revert this part of [your PR](https://github.com/odoo/odoo/pull/182196)? As there can be other this kind of cases like `active_id` in domain :thinking: 

Thank you!

Reply:
What you did should work in most cases. But please, check with security team if you change this.